src/dashboard/category-results-screen.js

```diff
@@ -15,8 +16,62 @@ function Paper ({ paper }) {
   ]);
 }
 
+function findRelativeDate (papers, range) {
+  const now = new Date();
+  const today = format(now, 'yyyy-MM-dd');
+  const displayPapers = [];
+
+  if (range === 'today') {
+    for (const paper of papers) {
+      const paperDate = format(paper.date, 'yyyy-MM-dd');
+      if (paperDate === today) {
+        displayPapers.push(paper);
```

Comment:
At the moment, all the categories seem to work as expected except for today's papers. When the data has papers from the day of, the `displayPapers` array here is still returned as empty. I'm currently looking into this, however, I've been having some issues with debugging since these functions are only run when the app is rendered in `app.js`. I've put up what I have at the moment in `launch.json` for some feedback, but perhaps I should be adding these date filtering functions in a new JS file? Not too sure, let me know your thoughts!

Reply:
See comment for  src/dashboard/categories.js